tests.js

```diff
@@ -170,5 +170,72 @@ describe('slugifyId()', function () {
   it('should do nothing if the id is undefined', function () {
     expect(kibiutils.slugifyId(undefined)).to.be(undefined);
   });
+});
+
+describe('doesQueryDependOnEntity()', function () {
+  var notDependentQueries = [
+      'select * \n' +
+       'from test \n' +
+       '-- where name = \'@doc[_source][github_id]@\''
+    ,
+      'select * \n' +
+       'from test \n' +
+        '/* where name \n' + '= \'@doc[_source][github_id]@\' */'
+    ,
+      'select * \n' +
+       'from test \n' +
+       '# where name = \'@doc[_source][github_id]@\''
+    ,
+      'select * \n' +
+       'from test \n' +
+        '/* where name \n' + '= \'@doc[_source] \n ' + '[github_id]@\ */'
+  ];
+
+  var DependentQueries = [
+     'select * \n' +
+       'from test \n' +
+       ' where name = \'@doc[_source][github_id]@\''
+    ,
+      'select * \n' +
+       'from test \n' +
+        '- where name = \'@doc[_source][github_id]@\''
```

Comment:
"I left incomplete for make sure regex is running properly, make sure regex looking beginning end finishing character for comment lines. Ok, I'm changing it now"
Same purpose , I'm changing it now too..